airflow/kubernetes/pod_generator.py

```diff
@@ -501,14 +505,18 @@ def reconcile_containers(base_containers,
         client_container = client_containers[0]
         base_container = base_containers[0]
         client_container = extend_object_field(
-            base_container,
-            client_container,
-            'volume_mounts',
-            'mount_path')
```

Comment:
Is this 'mount_path' removal deliberate?

Reply:
Hi @potiuk this removal is deliberate as that is how it is treated in 2.0. 2.0 only allows appends to lists. I believe this actually wroks out nicely as when there are multiple of hte same value k8s will just take the last value.

Reply:
Will that cause any more regression to the current behaviour from 1.10.11 / 1.10.12?

Reply:
@kaxil it shouldn't cause any regressions.